### ISSUE TITLE
Continuously logging "ERROR - Error in BLang parser rest service for composer"

### ISSUE THREAD

**Issue by dilinisg** [ballerinalang/composer#1028]
Pack - RC1
The below error error is continuously logged unnecessarily in the console.
Example: Write a code with a throw statement. When writing the throw key word (eg: throw error) this error is logged for each character typed.

```
2017-02-20 11:24:57,143 ERROR - Error in BLang parser rest service for composer

2017-02-20 11:24:58,730 ERROR - Error in BLang parser rest service for composer

2017-02-20 11:25:00,108 ERROR - Error in BLang parser rest service for composer
```

**Comment by nadeeshaan** [comment]
This issue has been fixed in the latest master. Hence closing the issue and if the issue can be still reproduced in the latest build, please reopen

**Comment by gnanakeethan** [comment]
I am getting the same error here. v0.89 Downloaded 20 mins ago.

![screenshot- 2017-07-03 at 8 11 47 am](https://user-images.githubusercontent.com/2353181/27776342-5d18fb6a-5fc7-11e7-9814-498c05fe4c62.jpg)

**Comment by nadeeshaan** [comment]
@gnanakeethan Are you getting this issue continuously? If so can you mention the steps to reproduce?

**Comment by gnanakeethan** [comment]
On a macOS system, I just downloaded this package. v0.89 from ballerinalang.com
1. Extracted the package
2. Ran the composer from the bin directory. (I have php composer in global. ) 
3. It resulted in the error.

**Comment by dakshika** [comment]
@nadeeshaan : I'm also getting same error on console

**Comment by nadeeshaan** [comment]
@dakshika @gnanakeethan we added this to track in the next release

**Comment by nadeeshaan** [comment]
Fixed in the from v0.94
